installer/sharch_body.sh

```diff
@@ -25,13 +25,19 @@ fi
 
 echo " OK."
 
+image_size=$(( $(sed -e '1,/^exit_marker$/d' "$0"  | tar --to-stdout -xf - | wc -c) / 1024))
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
 export cur_wd
 archive_path=$(realpath "$0")
 tmp_dir=$(mktemp -d)
 if [ "$(id -u)" = "0" ] ; then
     mount -t tmpfs tmpfs-installer $tmp_dir || exit 1
+    mount_size=$(df $tmp_dir | tail -1 | tr -s ' ' | cut -d' ' -f4)
+    if [ "$mount_size" -lt "$((image_size*3))" ]; then
```

Comment:
>image_size [](http://example.com/codeflow?start=31&length=10)

Is `image_sizes*1` is good enough?

Reply:
I have addressed the comment. Please review